src/ui/default_locale.js

```diff
@@ -15,8 +15,9 @@ const defaultLocale = {
     'ScaleControl.Meters': 'm',
     'ScaleControl.Kilometers': 'km',
     'ScaleControl.Miles': 'mi',
-    'ScaleControl.NauticalMiles': 'nm'
-
+    'ScaleControl.NauticalMiles': 'nm',
+    'ScrollZoomBlocker.CtrlMessage': 'ctrl + scroll to zoom the map',
```

Comment:
Does this need capitalization?

```suggestion
    'ScrollZoomBlocker.CtrlMessage': 'Ctrl + scroll to zoom the map',
```